integration_test/third_party_apps_data/applications/zookeeper/centos_rhel/install

```diff
@@ -0,0 +1,52 @@
+set -e
+
+sudo yum install -y curl java
+
+sudo mkdir -p /opt/zookeeper/stage
+sudo curl \
+    "https://dlcdn.apache.org/zookeeper/zookeeper-3.7.0/apache-zookeeper-3.7.0-bin.tar.gz" \
+    -o /opt/zookeeper/stage/zk.tgz
+sudo tar -xvzf /opt/zookeeper/stage/zk.tgz -C /opt/zookeeper --strip 1
+
+cat <<EOF | sudo tee /usr/lib/systemd/system/zookeeper.service
```

Comment:
Please do not override systemd units, especially in /usr/lib/systemd. If you need to customize something, you should use a systemd drop-in to set the one parameter you need: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.14.3

This presumably becomes unnecessary when you use the distro packages.

Reply:
This should not be overriding a systemd unit as it doesn't exist on the system by default. All instruction sets we've sound require adding this service file. There are almost no tutorials on using the package managers but I tested using them and none of them install the service file.

Reply:
This might be moot pending the other comment thread, but if you're writing your own unit it should go in `/etc/systemd/system` and not `/usr/lib/systemd`.

Reply:
Yeah.... not sure how I missed that thank you for pointing it out. I think I was caught up on the detail of just adding it at all.

Reply:
@quentinmit Changed this to `/etc/systemd/system`

Reply:
The install code looks good for a manual install now.